### DISCUSSION TITLE
Cannot read file node_modules/tsconfig.json

### DISCUSSION THREAD

**Discussion by akhilalekha** [facebook/docusaurus#3731] [Q&A]
I was getting this `./node_modules/@docusaurus/theme-classic/lib/theme/hooks/useDocs.js
Module not found: Can't resolve '@theme-init/hooks/useDocs'`.. warning so I just opened that folder to see what's going on. But out of nowhere this new error popped up and I haven't touched any file inside `node_modules`.

![image](https://user-images.githubusercontent.com/17964020/98946452-eae1aa80-2519-11eb-8a26-0a5682647e55.png)

**Comment by akhilalekha** [comment]
When I restarted it, this was gone

**Comment by RDIL** [comment]
This is just a vscode error, it shouldn't affect your site.
